src/server/scripts/Spells/spell_monk.cpp

```diff
@@ -159,9 +164,63 @@ class spell_monk_provoke : public SpellScript
     }
 };
 
+// Roll - 109132 or Roll (3 charges) - 121827
```

Comment:
Why does the comment mention 121827 when that spell is not given the script in database?